lib/datadog_backup/core.rb

```diff
@@ -73,19 +74,20 @@ def myclass
       self.class.to_s.split(':').last.downcase
     end
 
-    # Calls out to Datadog and checks for a '200' response
-    def update(id, body)
+    def create(body)
```

Comment:
What do you think if both `create` and `update` functions use the exact same `with_200` so we avoid duplicating retry logic. We can then add a new function here called `restore` (with same input as `update`) which makes the decision on which one to call. This will be the function which is called in cli.rb instead of `update`.